### PR TITLE
APIs and Models supporting workflow and tool details passed along to UI

### PR DESCRIPTION
**Pull request by dleehr**
- Adds Model/Serializer/View(s) for WorkflowVersionToolDetails - object that contains JSON describing the tools in a workflow, their docker images, package names, links, and versions.
- Adds detail route `GET /api/workflow-version/:id/version-info/` to fetch the contents of the `version_info_url` (e.g. CHANGELOG.md) and return wrapped in JSON.
- Hides logging output while running tests
- DRY: Refactors Admin API Test cases in `bespin_api_v2` to mix-in generic behavior (e.g. listing fails for unauthenticated users, admin can create, etc)

### CODE REVIEW

**Comment by johnbradley** [comment]
Was the detail route `/api/workflow-version/:id/version-info/` intentionally added to v1 and not v2 api?

File: bespin_api_v2/tests_api.py [hunk 1]
```diff
@@ -5,145 +5,181 @@
...
-        self.assertEqual(workflows[0].tag, 'exomeseq')
+        model_object = self.get_model_object(response.data['id'])
+        self.check_single_response(model_object, response.data)
 
     def test_put_fails_with_admin_user(self):
-        workflow = Workflow.objects.create(name='Exome Seq', tag='exomeseq')
         self.user_login.become_admin_user()
-        url = reverse('v2-admin_workflow-list') + str(workflow.id) + '/'
-        response = self.client.post(url, format='json', data={})
+        url = self.object_url(1)
```
Comment (johnbradley) [inline bespin_api_v2/tests_api.py]: The 1 key value here had me wondering if there was some code that created an item with that key. Reading the rest of this method makes sense that request is disallowed before that get's used.

File: bespin_api_v2/tests_api.py [hunk 2]
```diff
@@ -152,42 +188,11 @@ def test_create_with_admin_user(self):
...
-        self.user_login.become_admin_user()
-        url = reverse('v2-admin_workflowversion-list') + '1/'
-        response = self.client.put(url, format='json', data={})
-        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
-
-    def test_delete_fails_with_admin_user(self):
-        self.user_login.become_admin_user()
-        url = reverse('v2-admin_workflowversion-list') + '1/'
-        response = self.client.delete(url, format='json')
-        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
```
Comment (johnbradley) [inline bespin_api_v2/tests_api.py]: Extra blank line between these two methods.

**Review by johnbradley — APPROVED** [review]
Code LGTM. I like the admin api test refactoring. Only real question is about v1 only `/api/workflow-version/:id/version-info/` route I commented on earlier.

**Comment by dleehr** [comment]
> Was the detail route `/api/workflow-version/:id/version-info/` intentionally added to v1 and not v2 api?

Yes that was intentional - I only have plans to use that route from bespin-ui for now, which uses v1.

**Pull request merged**